www/source/tutorials/sample-app/next-steps.html.slim

```diff
@@ -16,10 +16,12 @@ section
     li Update configuration values dynamically to start the services up successfully
     li Upload your sample package to the depot
 
+  p If you want to extend the tutorial experience, dive into the #{link_to 'Using Habitat', '/docs/using-habitat/'} section of the docs to start playing with the many runtime features of Habitat. Or if you have a different web application that you would like to package, then we recommend you read through the #{link_to 'Build your Web App','/tutorials/build-your-own/'} guide. The guide is currently targeted at Ruby web applications and Node.js; however, if you are building with another framework such as Java web apps with Gradle, read #{link_to 'Scaffolding','/docs/glossary#glossary-scaffolding'} to find a scaffolding that can be used with your application type.
+
+  p Need more exercises? Learn Chef Rally has a series of tutorials focused on scaffolding, compiling applications from source, and packaging legacy applications in the #{link_to 'Build Applications with Habitat','https://learn.chef.io/tracks/habitat-build/'} track. In #{link_to 'Deploy Applications with Habitat','https://learn.chef.io/tracks/habitat-deploy/'} track you revisit Habitat Builder, update configuration dynamically, and setup service update strategies.
```

Comment:
We super appreciate the work going into the Learn Chef Rally tutorials but the IA structure and flow that we've settled upon is that once you're on the Habitat site you aren't linked back off to another Chef property. It's alright to send people from other Chef properties into the Habitat site, though! You're doing great work man 😄

Reply:
I forgot that by mentioned by Tasha many weeks back. I’ll recommit to this and remove the Learn Chef section.